inst/mappings/mapping_AgMIP.csv

```diff
@@ -0,0 +1,988 @@
+CONS.NRM;1000 t dm;Demand|Seed|Livestock products|Eggs;Mt DM/yr;1000;NULL;
+OTHU.NRM;1000 t dm;Demand|Seed|Livestock products|Eggs;Mt DM/yr;1000;NULL;
+CONS.NRM;1000 t dm;Demand|Seed|Livestock products|Monogastric meat;Mt DM/yr;1000;NULL;
+OTHU.NRM;1000 t dm;Demand|Seed|Livestock products|Monogastric meat;Mt DM/yr;1000;NULL;
+CONS.NRM;1000 t dm;Demand|Seed|Livestock products|Poultry meat;Mt DM/yr;1000;NULL;
+OTHU.NRM;1000 t dm;Demand|Seed|Livestock products|Poultry meat;Mt DM/yr;1000;NULL;
+CONS.RUM;1000 t dm;Demand|Seed|Livestock products|Ruminant meat;Mt DM/yr;1000;NULL;
+OTHU.RUM;1000 t dm;Demand|Seed|Livestock products|Ruminant meat;Mt DM/yr;1000;NULL;
+CONS.WHT;1000 t dm;Demand|Seed|Secondary products|Alcoholic beverages;Mt DM/yr;1000;NULL;
+OTHU.WHT;1000 t dm;Demand|Seed|Secondary products|Alcoholic beverages;Mt DM/yr;1000;NULL;
+CONS.CGR;1000 t dm;Demand|Seed|Secondary products|Brans;Mt DM/yr;1000;NULL;
+OTHU.CGR;1000 t dm;Demand|Seed|Secondary products|Brans;Mt DM/yr;1000;NULL;
+CONS.PFB;1000 t dm;Demand|Seed|Secondary products|Cotton lint;Mt DM/yr;1000;NULL;
+OTHU.PFB;1000 t dm;Demand|Seed|Secondary products|Cotton lint;Mt DM/yr;1000;NULL;
+CONS.CGR;1000 t dm;Demand|Seed|Secondary products|Distillers grains;Mt DM/yr;1000;NULL;
+OTHU.CGR;1000 t dm;Demand|Seed|Secondary products|Distillers grains;Mt DM/yr;1000;NULL;
+CONS.SGC;1000 t dm;Demand|Seed|Secondary products|Ethanol;Mt DM/yr;1000;NULL;
+OTHU.SGC;1000 t dm;Demand|Seed|Secondary products|Ethanol;Mt DM/yr;1000;NULL;
+CONS.SGC;1000 t dm;Demand|Seed|Secondary products|Microbial protein;Mt DM/yr;1000;NULL;
+OTHU.SGC;1000 t dm;Demand|Seed|Secondary products|Microbial protein;Mt DM/yr;1000;NULL;
+CONS.SGC;1000 t dm;Demand|Seed|Secondary products|Molasses;Mt DM/yr;1000;NULL;
+OTHU.SGC;1000 t dm;Demand|Seed|Secondary products|Molasses;Mt DM/yr;1000;NULL;
+CONS.OSD;1000 t dm;Demand|Seed|Secondary products|Oilcakes;Mt DM/yr;1000;NULL;
+OTHU.OSD;1000 t dm;Demand|Seed|Secondary products|Oilcakes;Mt DM/yr;1000;NULL;
+CONS.OSD;1000 t dm;Demand|Seed|Secondary products|Oils;Mt DM/yr;1000;NULL;
+OTHU.OSD;1000 t dm;Demand|Seed|Secondary products|Oils;Mt DM/yr;1000;NULL;
+CONS.SGC;1000 t dm;Demand|Seed|Secondary products|Sugar;Mt DM/yr;1000;NULL;
+OTHU.SGC;1000 t dm;Demand|Seed|Secondary products|Sugar;Mt DM/yr;1000;NULL;
+ECH4.AGR;MtCO2e;Emissions|CH4_GWP100AR6|Land|Agriculture;Mt CO2e/yr;1;NULL;
+ECH4.TOT;MtCO2e;Emissions|CH4_GWP100AR6|Land|Agriculture;Mt CO2e/yr;1;NULL;
+ECH4.MMG;MtCO2e;Emissions|CH4_GWP100AR6|Land|Agriculture|Animal waste management;Mt CO2e/yr;1;NULL;
+ECH4.ENT;MtCO2e;Emissions|CH4_GWP100AR6|Land|Agriculture|Enteric fermentation;Mt CO2e/yr;1;NULL;
+ECH4.RCC;MtCO2e;Emissions|CH4_GWP100AR6|Land|Agriculture|Rice;Mt CO2e/yr;1;NULL;
+ECH4.RIC;MtCO2e;Emissions|CH4_GWP100AR6|Land|Agriculture|Rice;Mt CO2e/yr;1;NULL;
+ECO2.TOT;MtCO2;Emissions|CO2|Land|Land-use Change;Mt CO2/yr;1;NULL;
+EMIS.TOT;MtCO2e;Emissions|GWP100AR6|Land;Gt CO2e/yr;1;NULL;
```

Comment:
Here I think the factor should be 1000, right? From million tonnes to giga tonnes.